File: app/src/main/java/com/semanientreprise/soundrecorderbox/MyAdapter.java
package com.semanientreprise.soundrecorderbox;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

public class MyAdapter extends FragmentPagerAdapter {
    private String[] fragment_titles = {"Record","Saved Recordings"};

    public MyAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        switch(position){
            case 0:{
                return RecordFragment.newInstance(position);
            }
            case 1:{
                return SavedRecordingsFragment.newInstance(position);
            }
        }
        return null;
    }

    @Override
    public int getCount() {
        return fragment_titles.length;
    }

    @Override
    public CharSequence getPageTitle(int position) {
        return fragment_titles[position];
    }
}

File: app/src/main/java/com/semanientreprise/soundrecorderbox/MyAdapter.kt
package com.semanientreprise.soundrecorderbox

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter

class MyAdapter(fm: FragmentManager?) : FragmentPagerAdapter(fm) {
    private val fragment_titles = arrayOf("Record", "Saved Recordings")
    override fun getItem(position: Int): Fragment {
        when (position) {
            0 -> {
                return RecordFragment.newInstance(position)
            }
            1 -> {
                return SavedRecordingsFragment.newInstance(position)
            }
        }
        return null
    }

    override fun getCount(): Int {
        return fragment_titles.size
    }

    override fun getPageTitle(position: Int): CharSequence {
        return fragment_titles[position]
    }
}